packages/react-storage/src/components/StorageBrowser/views/LocationActionView/utils.ts

```diff
@@ -0,0 +1,185 @@
+import { IconVariant } from '../../context/elements';
```

Comment:
Fwiw, we are trying to be more mindful in the JS repo about having "utils" or "helpers" files which tend to become dumping grounds for miscellaneous stuff and encourages modules that just keep growing. A particularly concerning issue is when the module has side effects and importing a some unrelated function from the module unintentionally triggers said side effect.

Having said all that, this is probably not the time or place to align on these philosophical differences.

Reply:
Yeah, I can split this out in my copy pr if it gets too big